src/Symfony/Component/HttpKernel/Profiler/FileProfilerStorage.php

```diff
@@ -61,7 +61,13 @@ public function find(?string $ip, ?string $url, ?int $limit, ?string $method, in
         $result = [];
         while (\count($result) < $limit && $line = $this->readLineFromFile($file)) {
             $values = str_getcsv($line);
-            [$csvToken, $csvIp, $csvMethod, $csvUrl, $csvTime, $csvParent, $csvStatusCode] = $values;
+            
+            try {
+                [$csvToken, $csvIp, $csvMethod, $csvUrl, $csvTime, $csvParent, $csvStatusCode] = $values;
+            } catch (\ErrorException $e) {
```

Comment:
This catch requires a specific error handler and we cannot assume that such an error handler is present at this point.

Don't catch `ErrorException`s. Prevent them from happening instead.

Reply:
Sure, I Can check if $values has 7 values and if not, continue ton the next iteration